src/client/components/SessionMoreInfo/SessionMoreInfoPopup.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import CloseIcon from "@mui/icons-material/Close";
+import "./SessionMoreInfo.css";
+
+export default function SessionMoreInfoPopup(props) {
+  const sessions = useSelector((state) => state.fetchSession);
+  const session = sessions.find((element) => element.groupId === props.sessionID);
```

Comment:
I had issues using === instead of ==, even if the value is same it would return false, just wanted to confirm it works. Might be because of ==='s rule: If both operands are objects, return true only if they refer to the same object.

Reply:
Weird, it seemed to be working for me